js/everything.js

```diff
@@ -84,6 +85,28 @@ function onPlayerReady(event) {
     // Setup player
     if (currentVideoID) {
         ZenPlayer.init(currentVideoID);
+        setupPlyr();
+    }
+}
+
+function setupPlyr() {
+    //set up Plyr player
```

Comment:
We should return here if `plyrPlayer` is truthy. Not sure what would happen if one manually invoked `setupPlyr()` several times on the console :smile: